### DISCUSSION TITLE
Are there any examples of actual use in a project?

### DISCUSSION THREAD

**Discussion by opvexe** [cue-lang/cue#1843] [Q&A]
Are there any examples of actual use in a project?

**Comment by verdverm** [comment]
These projects use CUE as core to their UX/DX: https://cuetorials.com/overview/whos-using-cue/

I must admit, I am disappointed in GitHub search, the get a fail [for this](https://github.com/search?q=language%3ACUE)

There are a number of projects starting to directly use CUE as an alternative to Yaml/JSON for config, like Caddy

Most people are using CUE primarily as a better Yaml/JSON, and then exporting back to those formats in all the places.

**Reply by slewiskelly** [reply]
> I must admit, I am disappointed in GitHub search, the get a fail [for this](https://github.com/search?q=language%3ACUE)

GitHub search is indeed extremely underwhelming. Sourcegraph recently began to index public repos, [here](https://sourcegraph.com/search?q=context:global+lang:CUE+select:repo&patternType=standard) is a list of repos that contain CUE files.

Note that searches for `lang:CUE` returns matches for `lang:Cue Sheet`, and vice versa.

**Reply by mvdan** [reply]
cs.github.com is more powerful than GitHub's regular search engine, I've come to find. For example: https://cs.github.com/?scopeName=All+repos&scope=&q=path%3A**%2Fcue.mod%2Fmodule.cue
